src/lib/features/events/event-store.ts

```diff
@@ -225,27 +234,11 @@ export class EventStore implements IEventStore {
             .from(TABLE)
             .where('id', '>', start)
             .andWhere('id', '<=', end)
-            .andWhere((builder) =>
-                builder
-                    .andWhere((inner) =>
-                        inner
-                            .whereNotNull('feature_name')
-                            .whereNotIn('type', [
-                                FEATURE_CREATED,
-                                FEATURE_TAGGED,
-                            ]),
-                    )
-                    .orWhereIn('type', [
-                        SEGMENT_UPDATED,
-                        FEATURE_IMPORT,
-                        FEATURES_IMPORTED,
-                        SEGMENT_CREATED,
-                        SEGMENT_DELETED,
-                    ]),
-            )
+            .andWhere(this.typeIsInteresting())
             .orderBy('id', 'asc');
 
         const rows = await query;
+        stopTimer();
```

Comment:
Missing stop timer means we were not receiving the metric and maybe leaking memory